### ISSUE TITLE
Sticker being removed from snap

### ISSUE THREAD

**Issue by MildPanda** [marzika/Snapprefs#241]
> Any Idea why when I put one of my stickers in the snap it doesn't appear when I post it to story. Was it too big or something? Tried a bunch of times to post it with it stretched out a bunch so its easily readable, then I tried smaller just once and it worked so would it be because of its size it maxed out the image cap so it just deleted the sticker? tried <<< that a couple times again and seemed to be fidgety still. (all of those attempts had additional text added to the snap)
> Deleted my text and just made the sticker by itself and humongous and that worked fine on the first try... perhaps just luck? Nope, tested a second time and it worked again.

That was my message from xda forums. Haven't messed around with the issue since. Please let me know what else I can do to help fix the issue.

**Comment by M1kep** [comment]
Can you upload the sticker your attempting to use and give some steps for us to attempt to re-produce?

**Comment by MildPanda** [comment]
@M1kep Surely. 
[1f3c3.zip](https://github.com/marzika/Snapprefs/files/496682/1f3c3.zip)
those are the original files I used. keeping it named the same thing should place that image in the 3rd to the last category (one with the guy that has his hands up) and for me its on the very upper left. 

I just tried to recreate the issue by doing what I had last time (although different background picture) which is text that goes onto the second line and the sticker **at least** 1/4th of the screen then upload to your story. First time I tried it didn't occur. My background picture was originally a picture of a scene from a netflix show (black list) and it showed the title of the show (so you'd need to pause the show for awhile and wait for that to show up) and the room was dark as it was night time. My assumption is the reason I couldn't get it to work the first try is because my background image wasn't high quality... idk. Thanks for the response!
